ethereum/spec/src/main/java/tech/pegasys/teku/spec/schemas/registry/SchemaRegistryBuilder.java

```diff
@@ -664,6 +691,81 @@ private static SchemaProvider<?> createSignedAggregateAndProofSchemaProvider() {
         .build();
   }
 
+  private static SchemaProvider<?> createCellSchemaProvider() {
+    return providerBuilder(CELL_SCHEMA)
+        .withCreator(
+            FULU,
+            (registry, specConfig, schemaName) ->
+                new CellSchema(SpecConfigFulu.required(specConfig)))
+        .build();
+  }
+
+  private static SchemaProvider<?> createDataColumnSchemaProvider() {
+    return providerBuilder(DATA_COLUMN_SCHEMA)
+        .withCreator(
+            FULU,
+            (registry, specConfig, schemaName) ->
+                new DataColumnSchema(SpecConfigDeneb.required(specConfig), registry))
+        .build();
+  }
+
+  private static SchemaProvider<?> createDataColumnSidecarSchemaProvider() {
+    return providerBuilder(DATA_COLUMN_SIDECAR_SCHEMA)
+        .withCreator(
+            FULU,
+            (registry, specConfig, schemaName) ->
+                DataColumnSidecarSchema.create(
+                    SignedBeaconBlockHeader.SSZ_SCHEMA,
+                    registry.get(DATA_COLUMN_SCHEMA),
+                    SpecConfigFulu.required(specConfig)))
+        .build();
+  }
+
+  private static SchemaProvider<?> createMatrixEntrySchemaProvider() {
+    return providerBuilder(MATRIX_ENTRY_SCHEMA)
+        .withCreator(
+            FULU,
+            (registry, specConfig, schemaName) ->
+                MatrixEntrySchema.create(registry.get(CELL_SCHEMA)))
+        .build();
+  }
+
+  private static SchemaProvider<?> createDataColumnSidecarsByRootRequestMessageSchemaProvider() {
+    return providerBuilder(DATA_COLUMN_SIDECARS_BY_ROOT_REQUEST_MESSAGE_SCHEMA)
+        .withCreator(
+            FULU,
+            (registry, specConfig, schemaName) ->
+                new DataColumnSidecarsByRootRequestMessageSchema(
+                    SpecConfigFulu.required(specConfig)))
+        .build();
+  }
+
+  private static SchemaProvider<?> createDataColumnSidecarsByRangeRequestMessageSchemaProvider() {
+    return providerBuilder(DATA_COLUMN_SIDECARS_BY_RANGE_REQUEST_MESSAGE_SCHEMA)
+        .withCreator(
+            FULU,
+            (registry, specConfig, schemaName) ->
+                new DataColumnSidecarsByRangeRequestMessage
+                    .DataColumnSidecarsByRangeRequestMessageSchema(
+                    SpecConfigFulu.required(specConfig)))
+        .build();
+  }
+
+  private static SchemaProvider<?> createMetadataMessageSchemaProvider() {
```

Comment:
Shouldn't it be used in Phase0 schema registry? It's now not used at all. And you could remove Altair override after it.

Reply:
I am not sure I understood your question. We are using this on `SchemaRegistryBuilder.create()` method, under the Phase 0 entries.